lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1885,3 +1980,250 @@ def _get_url(self, course_key, checkpoint):
+    """
+    Test email sending on re-verification
+    """
+    
+    def build_course(self):
+        """
+        Build up a course tree with a Reverificaiton xBlock.
+        """
+        # pylint: disable=attribute-defined-outside-init
+
+        self.course_key = SlashSeparatedCourseKey("Robot", "999", "Test_Course")
+        self.course = CourseFactory.create(org='Robot', number='999', display_name='Test Course')
+        self.due_date = datetime(2015, 6, 22, tzinfo=pytz.UTC)
+
+        # Create the course modes
+        for mode in ('audit', 'honor', 'verified'):
+            min_price = 0 if mode in ["honor", "audit"] else 1
+            CourseModeFactory(mode_slug=mode, course_id=self.course_key, min_price=min_price)
+
+        # Create the 'edx-reverification-block' in course tree
+        section = ItemFactory.create(parent=self.course, category='chapter', display_name='Test Section')
+        subsection = ItemFactory.create(parent=section, category='sequential', display_name='Test Subsection')
+        vertical = ItemFactory.create(parent=subsection, category='vertical', display_name='Test Unit')
+
+        self.reverification = ItemFactory.create(
+            parent=vertical,
+            category='edx-reverification-block',
+            display_name='Test Verification Block',
+            metadata={'attempts': 3, 'due': self.due_date}
+        )
+
+        self.section_location = section.location
+        self.subsection_location = subsection.location
+        self.vertical_location = vertical.location
+        self.reverification_location = self.reverification.location
+        self.assessment = "midterm"
+
+        self.re_verification_link = reverse(
+            'verify_student_incourse_reverify',
+            args=(
+                unicode(self.course_key),
+                unicode(self.assessment),
+                unicode(self.reverification_location)
+            )
+        )
+
+    def setUp(self):
+        super(TestEmailMessageWithCustomICRVBlock, self).setUp()
+        self.build_course()
+        self.check_point1 = VerificationCheckpoint.objects.create(
+            course_id=self.course.id, checkpoint_name=self.assessment
+        )
+
+        self.check_point1.add_verification_attempt(SoftwareSecurePhotoVerification.objects.create(user=self.user))
+
+        self.dummy_reverification_item_id_1 = self.reverification_location
+        VerificationStatus.add_verification_status(
+            checkpoint=self.check_point1,
+            user=self.user,
+            status='submitted',
+            location_id=self.dummy_reverification_item_id_1
+        )
+
+        self.attempt = SoftwareSecurePhotoVerification.objects.filter(user=self.user)
+
+    def test_approved_email_message(self):
+
+        subject, body = _compose_message_reverification_email(
+            self.course.id, self.user.id, "midterm", self.attempt, "approved", True
+        )
+
+        self.assertIn(
```

Comment:
For better readability change all the places where `assertIn` is used, to following:
```python
self.assertIn(
    "Your verification for course {course_name} and assessment {assessment} has been passed.".format(
        course_name=self.course.display_name_with_default,
        assessment=self.assessment
    ),
    body
)
```